<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FIT0201ANISUTINA_Plotter</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="cassiniTab">
       <attribute name="title">
        <string>Cassini</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="CassiniWidget" name="cassiniPlotWg" native="true"/>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="controlPanelCas">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>80</height>
           </size>
          </property>
          <property name="title">
           <string/>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="1" column="0">
            <widget class="QLabel" name="focus1Lbl">
             <property name="text">
              <string>focus1</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="focus2Lbl">
             <property name="text">
              <string>focus2</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="foc1X">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>20</width>
               <height>0</height>
              </size>
             </property>
             <property name="baseSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>3.6</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QLabel" name="foc2Y">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>40</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>2.14</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="foc1Y">
             <property name="minimumSize">
              <size>
               <width>20</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>5.44</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="foc2X">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>40</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>5.58</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QLabel" name="radiusValue">
             <property name="minimumSize">
              <size>
               <width>10</width>
               <height>0</height>
              </size>
             </property>
             <property name="layoutDirection">
              <enum>Qt::RightToLeft</enum>
             </property>
             <property name="text">
              <string>1.98</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QSlider" name="radiusSlider">
             <property name="minimum">
              <number>5</number>
             </property>
             <property name="maximum">
              <number>400</number>
             </property>
             <property name="value">
              <number>100</number>
             </property>
             <property name="sliderPosition">
              <number>100</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="curveTab">
       <attribute name="title">
        <string>Curve</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="CurveWidget" name="curvePlotWg" native="true">
          <property name="cursor">
           <cursorShape>CrossCursor</cursorShape>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>80</height>
           </size>
          </property>
          <property name="title">
           <string/>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QLabel" name="stepLabel">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>16</height>
              </size>
             </property>
             <property name="text">
              <string>step</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="stepValue">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>16</height>
              </size>
             </property>
             <property name="text">
              <string>25</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0" rowspan="2">
            <widget class="QPushButton" name="resetScaleButton">
             <property name="palette">
              <palette>
               <active>
                <colorrole role="WindowText">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>0</red>
                   <green>0</green>
                   <blue>0</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Button">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>210</red>
                   <green>240</green>
                   <blue>189</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Light">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>255</red>
                   <green>255</green>
                   <blue>255</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Midlight">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>232</red>
                   <green>247</green>
                   <blue>222</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Dark">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>105</red>
                   <green>120</green>
                   <blue>94</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Mid">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>140</red>
                   <green>160</green>
                   <blue>126</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Text">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>0</red>
                   <green>0</green>
                   <blue>0</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="BrightText">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>255</red>
                   <green>255</green>
                   <blue>255</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="ButtonText">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>0</red>
                   <green>0</green>
                   <blue>0</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Base">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>255</red>
                   <green>255</green>
                   <blue>255</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Window">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>210</red>
                   <green>240</green>
                   <blue>189</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Shadow">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>0</red>
                   <green>0</green>
                   <blue>0</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="AlternateBase">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>232</red>
                   <green>247</green>
                   <blue>222</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="ToolTipBase">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>255</red>
                   <green>255</green>
                   <blue>220</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="ToolTipText">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>0</red>
                   <green>0</green>
                   <blue>0</blue>
                  </color>
                 </brush>
                </colorrole>
               </active>
               <inactive>
                <colorrole role="WindowText">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>0</red>
                   <green>0</green>
                   <blue>0</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Button">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>210</red>
                   <green>240</green>
                   <blue>189</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Light">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>255</red>
                   <green>255</green>
                   <blue>255</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Midlight">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>232</red>
                   <green>247</green>
                   <blue>222</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Dark">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>105</red>
                   <green>120</green>
                   <blue>94</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Mid">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>140</red>
                   <green>160</green>
                   <blue>126</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Text">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>0</red>
                   <green>0</green>
                   <blue>0</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="BrightText">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>255</red>
                   <green>255</green>
                   <blue>255</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="ButtonText">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>0</red>
                   <green>0</green>
                   <blue>0</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Base">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>255</red>
                   <green>255</green>
                   <blue>255</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Window">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>210</red>
                   <green>240</green>
                   <blue>189</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Shadow">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>0</red>
                   <green>0</green>
                   <blue>0</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="AlternateBase">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>232</red>
                   <green>247</green>
                   <blue>222</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="ToolTipBase">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>255</red>
                   <green>255</green>
                   <blue>220</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="ToolTipText">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>0</red>
                   <green>0</green>
                   <blue>0</blue>
                  </color>
                 </brush>
                </colorrole>
               </inactive>
               <disabled>
                <colorrole role="WindowText">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>105</red>
                   <green>120</green>
                   <blue>94</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Button">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>210</red>
                   <green>240</green>
                   <blue>189</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Light">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>255</red>
                   <green>255</green>
                   <blue>255</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Midlight">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>232</red>
                   <green>247</green>
                   <blue>222</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Dark">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>105</red>
                   <green>120</green>
                   <blue>94</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Mid">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>140</red>
                   <green>160</green>
                   <blue>126</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Text">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>105</red>
                   <green>120</green>
                   <blue>94</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="BrightText">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>255</red>
                   <green>255</green>
                   <blue>255</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="ButtonText">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>105</red>
                   <green>120</green>
                   <blue>94</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Base">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>210</red>
                   <green>240</green>
                   <blue>189</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Window">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>210</red>
                   <green>240</green>
                   <blue>189</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Shadow">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>0</red>
                   <green>0</green>
                   <blue>0</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="AlternateBase">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>210</red>
                   <green>240</green>
                   <blue>189</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="ToolTipBase">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>255</red>
                   <green>255</green>
                   <blue>220</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="ToolTipText">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>0</red>
                   <green>0</green>
                   <blue>0</blue>
                  </color>
                 </brush>
                </colorrole>
               </disabled>
              </palette>
             </property>
             <property name="autoFillBackground">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>reset scale</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" rowspan="2">
            <widget class="QLabel" name="scaleLabel">
             <property name="text">
              <string>scale</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="scaleValue">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>16</height>
              </size>
             </property>
             <property name="text">
              <string>1.0</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CassiniWidget</class>
   <extends>QWidget</extends>
   <header>cassiniwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CurveWidget</class>
   <extends>QWidget</extends>
   <header>curvewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>radiusSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>radiusValue</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>486</y>
    </hint>
    <hint type="destinationlabel">
     <x>517</x>
     <y>486</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
